tests/core/training/story_reader/test_yaml_story_reader.py

```diff
@@ -0,0 +1,79 @@
+from rasa.core import training
+from rasa.core.events import ActionExecuted, UserUttered, SlotSet
```

Comment:
let's aim for 100% test coverage of the `YAMLReader`

Reply:
what about tests where markdown / yaml stories are mixed?

Reply:
I've added a new test file `test_common_story_reader.py`, and copied all example stories to the yaml.
It runs all the existing unit tests against both md and yaml now + there's a test that reads a mix of yaml and md.